src/gpuarray_blas_cuda_cublas.c

```diff
@@ -39,8 +39,8 @@ typedef struct _blas_handle {
 
 static const char *code_sgemvBH_N_a1_b1_small =                         \
   "extern \"C\"__global__ void sgemv(const float *A[], size_t lda, "    \
-  "                                  const float *x[], size_t incx, "   \
-  "                                  float *y[], size_t incy, "         \
+  "                                  const float *x[], int incx, "   \
+  "                                  float *y[], int incy, "         \
```

Comment:
Here you changed the data type in the kernel without changing the declared data type.  This will break badly.